qutebrowser/utils/usertypes.py

```diff
@@ -255,6 +256,47 @@ def reset(self):
 Backend = enum('Backend', ['QtWebKit', 'QtWebEngine'])
 
 
+# Where a download should be saved
+class DownloadTarget:
+
+    """Abstract base class for different download targets."""
+
+    def __init__(self):
+        raise NotImplementedError
+
+
+class FileDownloadTarget(DownloadTarget):
+
+    """Save the download to the given file.
+
+    Attributes:
+        filename: Filename where the download should be saved.
+    """
+
+    def __init__(self, filename):
```

Comment:
Hmm, I guess a `  # pylint: disable=super-init-not-called` here and in the other `__init__` methods makes sense here.